CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The flag `--es.ilm` has been renamed to `--collector.ilm`.
 * [CHANGE] Rename --es.data_stream to --collector.data-stream #983
 * [CHANGE] Rename --es.ilm to --collector.ilm #999
 
+## [Unreleased]
```

Comment:
There is already an unreleased section above this. Please use that section, following the format that already exists.